balance/balance_contract.go

```diff
@@ -272,8 +267,6 @@ func NewEpoch(epochNum int) bool {
 			token.transfer(ctx, addr, acc.Parent, acc.Balance, true, details)
 		}
 	}
-
-	return true
 }
 
 func Mint(to interop.Hash160, amount int, txDetails []byte) bool {
```

Comment:
`Mint` and `Burn` methods are not the part of NEP-17. Maybe it makes sense to remove return type there as well?

Reply:
Yes, wiil do this.